conceptnet5/readers/emoji.py

```diff
@@ -0,0 +1,49 @@
+import xml.etree.ElementTree as et
+from conceptnet5.edges import make_edge
+from conceptnet5.formats.msgpack_stream import MsgpackStreamWriter
+from conceptnet5.uri import Licenses, concept_uri
+
+def is_sentence(text):
+	if(' ' in text and not '|' in text):
+		return True
+	else:
+		return False
+
+def amt_words(text):
+	return text.count('|')+1
+
+def strip_words(text):
+	res =  text.split('|')
+	for i in range(len(res)):
+		res[i] = res[i].replace(" ","")
+	return res
+
+def handle_file(input_file, output_file):
+	tree = et.parse(input_file)
+	root = tree.getroot()
+	lang = root[0][1].attrib['type']
+	out = MsgpackStreamWriter(output_file)
+	for annotation in root[1]:
+		if(not is_sentence(annotation.text)):
```

Comment:
Now I could make more nitpicks about the formatting of code, but mostly I recommend that you `pip install flake8` and then run `flake8` on the Python file. This will tell you about likely errors in the code, as well as code formatting that breaks the [PEP 8](https://www.python.org/dev/peps/pep-0008/) standard.

There are times when it's justified to ignore PEP 8, but we usually try to follow its advice. If all committed code follows PEP 8, not only is it easier to read, but it prevents meaningless differences when the code is edited later.